assets/js/pages/ClusterDetails/HanaClusterDetails.stories.jsx

```diff
@@ -37,6 +52,13 @@ const hosts = [
   hostFactory.build({ hostname: details.nodes[1].name }),
 ];
 
+const scaleOutHosts = [
```

Comment:
What about using the map function here? 
`const scaleOutHosts = scaleOutDetails.nodes.map(node => hostFactory.build({ hostname: node.name }));
`